jax/interpreters/partial_eval.py

```diff
@@ -1818,7 +1816,11 @@ def process_call(self, call_primitive, f, explicit_tracers, params):
     in_tracers = [*implicit_tracers, *explicit_tracers]
     # TODO(mattjj): check in_tracers are consistent with f.in_type annotation
     with core.new_sublevel():
-      jaxpr, out_type, consts = trace_to_subjaxpr_dynamic2(f, self.main)
+      # TODO(lenamartens): fill out debug_info here.
```

Comment:
In a first iteration, I kept debug_info to None here, to be filled out later. This means in the nested case we don't get debug_info and leave that to a future change. However this regressed the `test_concrete_error_because_const` test cases, I think because the call to `jnp.add` then sets `debug_info` to None.

So I think we should either:
a) submit this and skip those tests, to be re-enabled later once we thread through debug_info properly
b) figure out how to populate debug_info here (maybe it's easy?). I tried, but couldn't really figure it out,

WDYT?